src/sockperf.cpp

```diff
@@ -102,6 +102,9 @@ static bool sock_lib_started = 0; //
 static int s_fd_max = 0;
 static int s_fd_min = 0; /* used as THE fd when single mc group is given (RECVFROM blocked mode) */
 static int s_fd_num = 0;
+#ifdef USING_VMA_EXTRA_API
+static int s_fd_max_zcopy = 0;
```

Comment:
Didn't find a counter for TCP accepted sockets.
This counter will include TCP accepted sockets, and used for zcopy scenario only.

Reply:
I think you do not need this variable as far as `g_zeroCopyData` is map and you can get number of element using `g_zeroCopyData.size()`.